web/package.json

```diff
@@ -114,7 +114,8 @@
     "moment-locales-webpack-plugin": "^1.2.0",
     "moment-timezone-data-webpack-plugin": "1.5.0",
     "raf": "^3.4.1",
-    "react-hot-loader": "^4.12.19",
```

Comment:
react-hot-loader is deprecated and did not work together with react-router-dom as it wraps every component in `ProxyFacade`